sw/device/silicon_creator/lib/cert/ot_certs.bzl

```diff
@@ -0,0 +1,127 @@
+# Copyright lowRISC contributors.
```

Comment:
This should go in `rules/` and perhap be called `rules/certificates.bzl` or something like that?

Reply:
Ok